core/arch/arm/include/sm/optee_smc.h

```diff
@@ -470,19 +470,19 @@
 	OPTEE_SMC_RPC_VAL(OPTEE_SMC_RPC_FUNC_FREE)
 
 /*
- * Deliver an IRQ in normal world.
+ * Deliver a foreign interrupt in normal world.
  *
  * "Call" register usage:
- * a0	OPTEE_SMC_RETURN_RPC_IRQ
+ * a0	OPTEE_SMC_RETURN_RPC_FOREIGN_INTR
```

Comment:
Btw, the changes here and in the rest of the file must also be changed in our kernel driver, i.e., https://github.com/linaro-swg/linux branch "optee".